### PR TITLE
Move ConfigurableFSGroupPolicy to GA

### PR DESCRIPTION
**Pull request by gnufied**
Add docs PR

xref https://github.com/kubernetes/enhancements/issues/695

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ different Kubernetes components.
 | `CSRDuration` | `true` | Beta | 1.22 | |
 | `ConfigurableFSGroupPolicy` | `false` | Alpha | 1.18 | 1.19 |
 | `ConfigurableFSGroupPolicy` | `true` | Beta | 1.20 | |
+| `ConfigurableFSGroupPolicy` | `true` | GA | 1.23 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:90]: Hi. I'm afraid this needs to move out of the section “Feature gates for Alpha or Beta features” and under the heading for graduated features.

**Review by sftim — CHANGES_REQUESTED** [review]
Hi. Thanks for this PR. I'm afraid it's not right and some small corrections are still needed.

Please update https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods to mark the feature as GA.

I've also added inline feedback about the right home for the feature-state information in the feature gates list.

**Comment by xing-yang** [comment]
I think we need to update this page as well. It says "beta".
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods

**Comment by xing-yang** [comment]
@gnufied Tomorrow is the deadline for doc to be reviewed and ready to merge. Can you address the comments?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (gnufied) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: fixed.

**Comment by xing-yang** [comment]
The content looks good.
I see some CI failures.

**Comment by xing-yang** [comment]
@gnufied Can you squash the commits?

**Comment by xing-yang** [comment]
The content looks good.
/lgtm

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (jlbutler) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: confirming this requested change landed.

**Pull request merged**